spec/mongo/cluster/topology_spec.rb

```diff
@@ -105,28 +105,6 @@
           expect(topology).to be_a(Mongo::Cluster::Topology::Unknown)
         end
       end
-
```

Comment:
What's the rationale for removing these tests?

Reply:
As mentioned in the commit these tests are duplicated.